Formula/xcbuild.rb

```diff
@@ -0,0 +1,21 @@
+class Xcbuild < Formula
+  desc "Xcode-compatible build tool"
+  homepage "https://github.com/facebook/xcbuild"
+  url "https://github.com/facebook/xcbuild.git", :tag => "0.1.0", :revision => "08575f65f1c907e280ed4663b5461b7cfaf5cfaa"
+  head "https://github.com/facebook/xcbuild.git", :shallow => false
+
+  depends_on "ninja"
+  depends_on "cmake"
+  depends_on "libpng"
+  depends_on "pkg-config"
+
+  def install
+    system "cmake", ".", "-G", "Ninja", *std_cmake_args
+    system "ninja", "install"
+    bin.install_symlink "#{prefix}/usr/bin/xcbuild"
+  end
+
+  test do
+    system "#{bin}/xcbuild", "-showsdks"
```

Comment:
Not really, as it is dependant on what SDKs the user has installed on their machine.

For instance mine is:
```
$ xcbuild -showsdks
iOS SDKs:
	iOS 10.2                        -sdk iphoneos10.2
	iOS 10.2                        -sdk iphoneos10.2

iOS Simulator SDKs:
	Simulator - iOS 10.2            -sdk iphonesimulator10.2
	Simulator - iOS 10.2            -sdk iphonesimulator10.2

macOS SDKs:
	macOS 10.12                     -sdk macosx10.12
	macOS 10.12                     -sdk macosx10.12

tvOS SDKs:
	tvOS 10.1                       -sdk appletvos10.1
	tvOS 10.1                       -sdk appletvos10.1

tvOS Simulator SDKs:
	Simulator - tvOS 10.1           -sdk appletvsimulator10.1
	Simulator - tvOS 10.1           -sdk appletvsimulator10.1

watchOS SDKs:
	watchOS 3.1                     -sdk watchos3.1
	watchOS 3.1                     -sdk watchos3.1

watchOS Simulator SDKs:
	Simulator - watchOS 3.1         -sdk watchsimulator3.1
	Simulator - watchOS 3.1         -sdk watchsimulator3.1```